Add eyeglass support

ahmedelgabri opened the pull request:
Since you already support `node-sass`, adding [eyeglass](https://github.com/sass-eyeglass/eyeglass) support is easy, this will make dealing with Sass files using `npm` easier.

Users now can do `npm install --save sass-mq` and then inside their project they just import the file as usual like so.

```sass
@import 'sass-mq';

.responsive {
    // Apply styling to mobile and upwards
    @include mq($from: mobile) {
        color: red;
    }
    // Apply styling up to devices smaller than tablets (exclude tablets)
    @include mq($until: tablet) {
        color: blue;
    }
    // Same thing, in landscape orientation
    @include mq($until: tablet, $and: '(orientation: landscape)') {
        color: hotpink;
    }
    // Apply styling to tablets up to desktop (exclude desktop)
    @include mq(tablet, desktop) {
        color: green;
    }
}
```

Since you already going to add `npm` support in #52 I thought this might be helpful too.

_The readme update is silly, so I'm open on how to improve this._

**Note** I didn't want to change the project file/folder structure that's why I didn't follow the recommendations of eyeglass. If you are fine with changing the file/folder structure. I can update the PR.

kaelig commented on README.md:
```diff
@@ -12,12 +12,9 @@ and is now also in use at BBC Sport and the Financial Times…_
 
 ## How to Use It
 
-Immediately play with it on [SassMeister](http://sassmeister.com/): `@import 'mq';`.
```
Can you please restore this? It was added yesterday.

kaelig commented on README.md:
```diff
@@ -12,12 +12,9 @@ and is now also in use at BBC Sport and the Financial Times…_
-
 1. Install with [Bower](http://bower.io/ "BOWER: A package manager for the web"):
    `bower install sass-mq --save`
+   OR Install with [npm](https://www.npmjs.com/): `npm install "sass-mq/sass-mq" --save` _it supports [eyeglass](https://github.com/sass-eyeglass/eyeglass)_
```
Let's put each of these steps on a new line (it renders as one line only).

ahmedelgabri replied:
But it doesn't need an extra installation step, maybe add this as a feature _that it also works as an eyeglass module_?

kaelig replied:
Sorry I  meant it should look like this:

1. Install with [Bower](http://bower.io/ "BOWER: A package manager for the web"):
   `bower install sass-mq --save`  
   OR Install with [npm](https://www.npmjs.com/): `npm install "sass-mq/sass-mq" --save` _it supports [eyeglass](https://github.com/sass-eyeglass/eyeglass)_  
   OR [Download _mq.scss](https://raw.github.com/sass-mq/sass-mq/master/_mq.scss)
   to your Sass project.
2. Import the partial in your Sass files and override default settings

```md
1. Install with [Bower](http://bower.io/ "BOWER: A package manager for the web"):
   `bower install sass-mq --save`  
   OR Install with [npm](https://www.npmjs.com/): `npm install "sass-mq/sass-mq" --save` _it supports [eyeglass](https://github.com/sass-eyeglass/eyeglass)_  
   OR [Download _mq.scss](https://raw.github.com/sass-mq/sass-mq/master/_mq.scss)
   to your Sass project.
```

ahmedelgabri replied:
No problem. Will update the `README`

kaelig commented on package.json:
```diff
@@ -0,0 +1,55 @@
+    "test.sh"
+  ],
+  "files": [
+    "_mq.scss",
```
Should the eyeglass manifest be included in the list of files?

ahmedelgabri replied:
Yes, I'll add it to `files`.

kaelig commented:
Thanks, this is a great idea.

Can you please add a test to the travis file that:

1. npm installs node-sass, eyeglass, and sass-mq
2. compiles an example file (just to make sure it can be imported properly)

Cheers!

kaelig commented:
> Note I didn't want to change the project file/folder structure that's why I didn't follow the recommendations of eyeglass. If you are fine with changing the file/folder structure. I can update the PR.

How about creating `sass/index.scss` with these contents:

```scss
// This file is for eyeglass 
@import '../mq';
```

Then I guess this should work when using sass-mq as an eyeglass module?

```scss
@import 'sass-mq';
```

ahmedelgabri commented:
Yes, I assume it should work.

BTW, there is a minor issue with [eyelglass](https://github.com/sass-eyeglass/eyeglass-sample/issues/1#issuecomment-109384519) right now that it doesn't always pick up modules form `devDependencies` but will always pick it up form `dependencies`, maybe we should also add a note for it in the README too?

ahmedelgabri commented:
@kaelig I have updated the PR and added a test _which runs locally fine_ I tried to fix [travis](https://travis-ci.org/sass-mq/sass-mq/builds/66428676#L265-L275) but it didn't work any clue how to do this?
